packages/paste-core/layout/stack/__tests__/stack.test.tsx

```diff
@@ -99,4 +110,9 @@ describe('Stack', () => {
     const {asFragment} = testRender(<MockResponsiveStack />);
     expect(asFragment()).toMatchSnapshot();
   });
+
+  it('should render as a header', () => {
+    const {asFragment} = testRender(<MockHeaderStack />);
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
I think for these I would like you to assert the element outside of a snapshot. And also provide some tests for arbitary html attributes

Reply:
I changed that test and added a couple more for valid attributes and one for the invalid className attribute. Let me know if I should add more.